include/wx/msw/window.h

```diff
@@ -734,10 +734,23 @@ class WXDLLIMPEXP_CORE wxWindowMSW : public wxWindowBase
     // this function is called after the window was resized to its new size
     virtual void MSWEndDeferWindowPos()
     {
+        // fix position if limited by Short range
```

Comment:
This function should arguably be moved into `src/msw/window.cpp` now that it's not really trivial any more.

Reply:
Yes, sorry for not making it clear.